public/js/utils.jsx

```diff
@@ -158,3 +158,12 @@ export function validateEmailAsYouType(email) {
     };
   }
 }
+
+export function hasErrorSubString(errorList, searchSubString) {
```

Comment:
you could just call this `arrayHasSubString()` because it seems like it could be used for any array.